engine/execution/computation/execution_verification_test.go

```diff
@@ -674,10 +672,10 @@ func executeBlockAndVerifyWithParameters(t *testing.T,
 	ledgerCommiter := committer.NewLedgerViewCommitter(ledger, tracer)
 
 	bservice := requesterunit.MockBlobService(blockstore.NewBlockstore(dssync.MutexWrap(datastore.NewMapDatastore())))
-	trackerStorage := new(mocktracker.Storage)
-	trackerStorage.On("Update", mock.Anything).Return(func(fn exedatatracker.UpdateFn) error {
-		return fn(func(uint64, ...cid.Cid) error { return nil })
-	})
+	trackerStorage := mocktracker.NewMockStorage()
+
+	// Assume SetFulfilledHeight succeeds unless otherwise indicadted in a test requriement
+	trackerStorage.On("SetFulfilledHeight", mock.Anything).Return(nil)
```

Comment:
this line is also run in `mocktracker.NewMockStorage()` so you can probably skip it here